scripts/exwafs_grib2_0p25_blending.sh

```diff
@@ -2,152 +2,104 @@
-    echo "*** WARNING! Missing UK data for WAFS GRIB2 0P25 blending ***" >>mailmsg
+    echo "*** WARNING! Missing US data for WAFS GRIB2 0P25 blending ***" >>mailmsg
     echo "*************************************************************" >>mailmsg
     echo >>mailmsg
     echo "Send alert message to AWC ...... " >>mailmsg
     echo >>mailmsg
-    echo "One or more UK WAFS GRIB2 files not found:" >>mailmsg
-    cat missing_uk_files >>mailmsg
-    echo "Skipping t${cyc}z f${fhr}..." >>mailmsg    
-    cat mailmsg >"${COMOUT}/${RUN}.t${cyc}z.f${fhr}.wafs_blend_0p25_usonly.emailbody"
-    cat "${COMOUT}/${RUN}.t${cyc}z.f${fhr}.wafs_blend_0p25_usonly.emailbody" | mail.py -s "${subject}" "${MAILTO}" -v
-
-    # Distribute US WAFS unblend Data to NCEP FTP Server (WOC) and TOC
-    echo "altering the unblended US WAFS products:"
-    echo " - ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2"
-    echo " - ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx"
-
-    if [[ "${SENDDBN}" == "YES" ]]; then
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_UBL_GB2 "${job}" "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2"
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_UBL_GB2_WIDX "${job}" "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx"
-    fi
+    for file in $missing_us_files ; do
+        cat $file >>mailmsg
+    done
 
-else
-    # Distribute US WAFS unblend Data to NCEP FTP Server (WOC) and TOC
-    if [[ "${SENDCOM}" == "YES" ]]; then
-        cpfs "0p25_blended_${PDY}${cyc}f${fhr}.grib2" "${COMOUT}/WAFS_0p25_blended_${PDY}${cyc}f${fhr}.grib2"
-    fi
+    cat mailmsg >"${COMOUT}/${RUN}.t${cyc}z.wafs_blend_0p25_usmissing.emailbody"
+    cat "${COMOUT}/${RUN}.t${cyc}z.wafs_blend_0p25_usmissing.emailbody" | mail.py -s "${subject}" "${MAILTO}" -v
+fi
 
-    if [[ "${SENDDBN_NTC}" == "YES" ]]; then
-        #   Distribute Data to NCEP FTP Server (WOC) and TOC
-        echo "No WMO header yet"
-    fi
+if [[ ! -z "$no_blending_files" ]] ; then
+    subject="WARNING! Not blended for WAFS GRIB2 0P25 blending, ${PDY} t${cyc}z ${job}"
+    echo "*************************************************************" >mailmsg
+    echo "*** WARNING! Not blended for WAFS GRIB2 0P25 blending     ***" >>mailmsg
+    echo "*************************************************************" >>mailmsg
+    echo >>mailmsg
+    echo "Send alert message to AWC ...... " >>mailmsg
+    echo >>mailmsg
+    for file in $no_blending_files ; do
+        cat $file >>mailmsg
+    done
 
-    if [[ "${SENDDBN}" == "YES" ]]; then
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_BL_GB2 "${job}" "${COMOUT}/WAFS_0p25_blended_${PDY}${cyc}f${fhr}.grib2"
-    fi
+    cat mailmsg >"${COMOUT}/${RUN}.t${cyc}z.wafs_blend_0p25_noblending.emailbody"
+    cat "${COMOUT}/${RUN}.t${cyc}z.wafs_blend_0p25_noblending.emailbody" | mail.py -s "${subject}" "${MAILTO}" -v
 fi
+
+export err=$? ; err_chk
```

Comment:
If you reach here, `err` will almost be guaranteed to be 0.
```suggestion
err_chk
```